From fd882624af9f242c95bb67809bae2c8f5bf2abfa Mon Sep 17 00:00:00 2001
From: Alessandro Ghedini <alessandro@ghedini.me>
Date: Wed, 11 Oct 2017 15:43:50 -0700
Subject: [PATCH 1/3] SSL: include <openssl/hmac.h>.

This header carries the definition of HMAC_Init_ex(). In OpenSSL this
header is included by <openssl/ssl.h>, but it's not so in BoringSSL.

It's probably a good idea to explicitly include this header anyway,
regardless of whether it's included by other headers or not.
---
 src/event/ngx_event_openssl.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/event/ngx_event_openssl.h b/src/event/ngx_event_openssl.h
index b9a3a965..623d851d 100644
--- a/src/event/ngx_event_openssl.h
+++ b/src/event/ngx_event_openssl.h
@@ -22,6 +22,7 @@
 #include <openssl/engine.h>
 #endif
 #include <openssl/evp.h>
+#include <openssl/hmac.h>
 #ifndef OPENSSL_NO_OCSP
 #include <openssl/ocsp.h>
 #endif
-- 
2.15.0

